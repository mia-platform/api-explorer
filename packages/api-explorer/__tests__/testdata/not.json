{
    "components": {
        "withNot": {
            "type": "array",
            "items": {
              "not": {
                "type": "object",
                "properties": {}
              }
            }
        }
    },
    "$ref": "#/components/withNot"
}