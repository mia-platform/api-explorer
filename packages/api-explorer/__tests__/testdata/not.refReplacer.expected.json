{
    "definitions": {
        "components": {
            "definitions": {
                "withNot": {
                    "type": "array",
                    "items": {
                      "not": {
                        "type": "object",
                        "properties": {}
                      }
                    }
                }
            }
        }
    },
    "$ref": "#/definitions/components/definitions/withNot"
}